bootstrap.ps1

```diff
@@ -0,0 +1,20 @@
+param(
+    [switch]$build = $false
+)
+
+# Make sure we're in the repository root.
+cd "$PSScriptRoot"
+
+git fetch --tags --force --quiet
+
+$version = $( git tag --sort=committerdate | Select-Object -Last 1 )
+
+if ($build) {
+    git checkout $version
```

Comment:
I'm not sure on PowerShell quoting rules - does this need to be quoted?

Reply:
I'm not sure. But I tested it does work and there won't be special characters because we control the file name.